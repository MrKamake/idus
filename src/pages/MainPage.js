import React from 'react';
import { Link } from 'react-router-dom';
import './MainPage.scss'

const MainPage = () => {
  return (
    <div className="Main-Page">
      <Link to="/card">Card</Link>
      <Link to="/input">Input</Link>
    </div>
  );
};

export default MainPage;