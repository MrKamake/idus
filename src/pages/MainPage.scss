.Main-Page {
  padding-top: 15%;
  width: 90%;
  a {
    display: inline-block;
    margin: 5%;
    padding: 20% 30%;
    width: 40%;
    text-align: center;
    font-size: 1.5rem;
    text-decoration: none;
    color: white;
    background-color: gray;
    border-radius: 12px;
  }
}
