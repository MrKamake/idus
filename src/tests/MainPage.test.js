import React from 'react';
import { shallow } from 'enzyme';
import MainPage from '../pages/MainPage';

describe('Main Pgae', () => {
  describe('<MainPage />', () => {
    let wrapper;
    beforeEach(() => {
      wrapper = shallow(<MainPage />);
    });

    it('matches snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });

    it('should contain elements', () => {
      const MainPageComponent = wrapper.find('.Main-Page');
      expect(MainPageComponent.length).toBe(1);
    });
  });
});
