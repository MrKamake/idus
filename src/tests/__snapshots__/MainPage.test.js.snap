// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Pgae <MainPage /> matches snapshot 1`] = `
<div
  className="Main-Page"
>
  <Link
    to="/card"
  >
    Card
  </Link>
  <Link
    to="/input"
  >
    Input
  </Link>
</div>
`;
